requirements.txt

```diff
@@ -2,7 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy >=1.17.2
-torch >=1.8.1, <=1.13.1
+torch >=1.8.1, <=2.0.0
```

Comment:
Shouldn't we also bump max version for torchvision?

Reply:
good catch :)